src/lib/task.js

```diff
@@ -194,9 +194,9 @@ export class Reclaimer {
 
     this.claimTimeoutId =
       setTimeout(function() {
```

Comment:
This code is already in a `setTimeout` with no return, so I don't think async is doing anything here:

```js
this.claimTimeoutId = setTimeout(() => this.reclaimTask(), nextClaim);
```

If for some reason it is needed, should be easy as:

```js
this.claimTimeoutId = setTimeout(async () => await this.reclaimTask(), nextClaim);
```